kotti/resources.py

```diff
@@ -655,6 +660,27 @@ class SaveDataMixin(object):
         See http://stackoverflow.com/questions/30433960/how-to-use-declare-last-in-sqlalchemy-1-0  # noqa
     """
 
+    #: The filename is used in the attachment view to give downloads
+    #: the original filename it had when it was uploaded.
+    #: (:class:`sqlalchemy.types.Unicode`)
+    filename = Column(Unicode(100))
+    #: MIME type of the file
+    #: (:class:`sqlalchemy.types.String`)
+    mimetype = Column(String(100))
+    #: Size of the file in bytes
+    #: (:class:`sqlalchemy.types.Integer`)
+    size = Column(Integer())
+
+    #: Filedepot mapped blob
+    #: (:class:`depot.fileds.sqlalchemy.UploadedFileField`)
+    @declared_attr
+    def data(cls):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [An attribute defined in kotti.resources line 753 hides this method](https://www.codacy.com/app/disko/Kotti/file/1578056223/issues/source?bid=2089747&fileBranchId=3023493#l677)